src/player.cpp

```diff
@@ -3392,8 +3393,8 @@ int player::print_aim_bars( WINDOW *w, int line_number, item *weapon, Creature *
     // This simplifies the calculation greatly, that's intentional.
     const std::array<std::pair<double, char>, 3> ratings =
         {{ std::make_pair(0.1, '*'), std::make_pair(0.4, '+'), std::make_pair(0.6, '|') }};
-    const std::string confidence_label = _("Confidence: ");
```

Comment:
I could turn those extents into square brackets instead, which should be clearer visually. I could then either restore the colon to the original position, or remove it (as the leftmost bracket should be clearly seperating label from bar).